packages/@groupby/elements-gb-tracker-plugin/src/gb-tracker-plugin.ts

```diff
@@ -0,0 +1,98 @@
+import { Plugin, PluginMetadata, PluginRegistry } from '@groupby/elements-core';
+import { BEACON_SEARCH, BeaconSearchPayload } from '@groupby/elements-events';
+// eslint-disable-next-line import/no-unresolved
+import { AutoSearchEvent } from 'gb-tracker-client/models';
+// eslint-disable-next-line import/no-unresolved
+import { GbTracker } from 'gb-tracker-client/slim-es';
+
+/**
+ * This plugin is responsible for exposing an [[GbTracker]] instance to Core.
+ */
+export default class GbTrackerPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'gb-tracker',
+      depends: ['dom_events'],
+    };
+  }
+
+  /**
+   * The value that the [[GbTrackerPlugin]] exposes to the Core entity.
+   */
+  gbTracker: ReturnType<typeof GbTracker>;
+
+  /**
+   * A reference to the registry of plugins for later use.
+   */
+  core: PluginRegistry;
+
+  /**
+   * Name of the events plugin.
+   */
+  eventsPluginName = 'dom_events';
```

Comment:
I'm okay with this, although it's worth nothing we'll be pushing a greater inconsistency because three of our plugins still use this convention. I agree that it's not super useful, though.